apps/judicial-system/web/src/routes/Court/Indictments/CourtRecord/CourtSessionAccordionItem.tsx

```diff
@@ -211,7 +201,9 @@ const CourtSessionAccordionItem: FC<Props> = (props) => {
     patchSession,
   ])
 
-  useOnceOn(isCaseUpToDate, initialize)
+  // Initialize when the case is up to date and the accordion item is expanded
+  useOnceOn(isCaseUpToDate, () => setReadyForInitialization(true))
+  useOnceOn(readyForInitialization && isExpanded, initialize)
```

Comment:
Initialization is a bit tricky in subcomponents. This guaranties that the initialization is not perfermed before the case has been refreshed after navigating to the current page and that the initialization in only performed when the current court session is expanded. This prevents unnecessary update spams to the backend.